presto-docs/src/main/sphinx/release/release-0.273.rst

```diff
@@ -0,0 +1,82 @@
+=============
+Release 0.273
+=============
+
+**Highlights**
+==============
+* Add new Presto connector for Clickhouse.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix disaggregated coordinator resource group aggregation issue related to ``WAITING_FOR_PREREQUISITE`` queries. When a query is moved to WAITING_FOR_PREREQUISITE, resource manager would end up counting them towards running queries. This gives a wrong impression to coordinator about running queries and it would end up running less queries than configured.
+* Fix a bug where ``EXPLAIN (TYPE IO)`` did not include the index source from an index join.
+* Fix a memory over-counting issue for using varchar types that could cause queries to fail with exceeded memory limit errors.
+* Fix an off-by-one error during buffer size calculation in :func:`tdigest_agg`.
+* Fix memory tagging for UnnestOperator and TableWriterMergeOperator.
+* Improve performance of :func: `zip_with` when used inside a :func:`reduce_agg`.
+* Add :func:`to_base32` and :func:`from_base32` varbinary functions.
+* Add support for ``TRUNCATE TABLE`` statement and associated :doc:`/sql/truncate`.
+* Add support to set timeout on the query analyzer runtime to prevent long running query analysis like complex regular expressions. The timeout duration can be set by the configuration property ``planner.query-analyzer-timeout`` or session property ``query_analyzer_timeout``.
+* Add :func:`trim_array` function that can be called to delete elements from the end of an ordinary array.
+* Add a new optimization for showing results for (interactive) distinct limit N as they become available with no buffering.  This can be enabled by setting the session parameter: ``quick_distinct_limit_enabled`` to true.
```

Comment:
nit: ....by  setting the session **property** ``quick_distinct_limit_enabled`` to true.